yaml_files/cuba.yml

```diff
@@ -565,3 +555,59 @@ CUBA_KEYS:
     definition: Thermodynamic ensemble
     shape: [1]
     type: string
+
+  LATTICE_UC_ABC:
+    definition: Lattice translation vectors lengths (a, b, c) of an atomic unit cell
+    shape: [3]
+    type: double
+
+  LATTICE_UC_ANGLES:
+    definition: Alpha, Beta and Gamma angles of an atomic unit cell
+    shape: [3]
+    type: double
+
+  SYMMETRY_GROUP:
+    definition: Symmetry group number acording to the Volume A of International Tables for Crystallography
+    shape: [1]
+    type: integer
+
+  CRYSTAL_ORIENTATION_1:
+    definition: Crystallographic orientation defined by Miller Index of the reference surface
+    shape: [3, 3]
+    type: double
+
+  CRYSTAL_ORIENTATION_2:
+    definition: Crystallographic orientation defined by Miller Index of the non-reference surface
+    shape: [3, 3]
+    type: double
+
+  SHAPE_ORIENTATION_1:
+    definition: Real space orientation of a component. Shape-center system reference set to global cartesian system reference.
+    shape: [1, 3]
+    type: double
+
+  SHAPE_ORIENTATION_2:
+    definition: Real space orientation of a component. Shape-center system reference set to global cartesian system reference.
+    shape: [1, 3]
+    type: double
+
+  FILE_STL:
+    definition: Absolute path of a STL file
+    shape: [80]
+    type: string
+
+  STL_MODE:
+    definition: STL cutting process mode. Normal or negative cutting
+    shape: [1]
+    type: integer
+
+  STL_SCALING:
+    definition: Scaling parameter for components with STL shape
+    shape: [1]
+    type: double
+
+  STL_PADDING:
+    definition: Padding width in -X, +X, -Y, +Y, -Z and +Z directions for components with STL shape
+    shape: [6]
+    type: double
+
```

Comment:
As Goyo has remarked, all these CUBAs are nCAD attributes. They are used to poisoned an STL drawn in the atomic reference system:
- STL_MODE (0/1): 0 means that nCAD will use the STL drawn directly and will fill with atoms the shape contained by the triangles of the STL. 1 means that nCAD will build another STL shape that it is a cubic shape where the original STL is removed. A clear example is a complex assembly of channels or a porous like material with this hollow channels inside. 
- STL_Scaling: apply an scale factor to the STL. Hence, a user can use a STL created by traditional CAD (where you are using mm, microns, meters, ...) and imported it to nCAD in nanometric scale. 
- STL_Padding: It is used to move and set your 0 in the Unit cell and therefore translate the atomic structure distances minor than the UC